### PR TITLE
Add basic JDBC tests with Postgres

### PR DESCRIPTION
**Pull request by suyograo**
while I wait for the DNS issues to be resolved

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,7 +9,9 @@ rvm:
 jdk:
   - oraclejdk8
 env:
-  - INTEGRATION=true  
+  - INTEGRATION=true
+services:
+  - postgresql
```
Comment (talevy) [inline .travis.yml]: this is the first fixture that requires travis, right?

**Comment by talevy** [comment]
test is failing saying that the config syntax is invalid. I don't see anything outlandish

**Comment by talevy** [comment]
heya @suyograo, now that we have a path to running travis-ci locally (with these services), I am good with these changes. want to rebase?

**Comment by ph** [comment]
Rebase needed?

**Pull request closed without merging**